src/test/java/org/jenkinsci/plugins/workflow/cps/FlowDurabilityTest.java

```diff
@@ -251,6 +251,20 @@ static void verifySucceededCleanly(Jenkins j, WorkflowRun run) throws Exception
         assertHasTimingAction(run.getExecution());
     }
 
+    /** A minimal check of resuming the job with no node checks, in case of a dirty shutdown.
+     * Currently dirty shutdowns have the potential to lose the head node ID if they happen at the moment of writing
+     * it to build.xml
+     */
+    static void verifyDirtyResumed(JenkinsRule rule, WorkflowRun run, String logStart) throws Exception {
```

Comment:
```suggestion
    private static void verifyDirtyResumed(JenkinsRule rule, WorkflowRun run, String logStart) throws Exception {
```

Reply:
I copied that signature off `verifySafelyResumed` as it's a much simplified replacement for that

Reply:
Maybe so, I am just reviewing and seeing a newly introduced method referred to only within the same class, so it should be `private` because it can. Friendlier to IDEs, human readers, etc. to not have to look around for other references.

Reply:
shall we do the same for `verifySafelyResumed`?

Reply:
If you like.